src/Hero6.DesktopGL/Hero6.DesktopGL.csproj

```diff
@@ -15,6 +15,7 @@
     <MonoGameContentBuilderExe Condition=" '$(OS)' != 'Windows_NT' And Exists ('/opt/monogame-pipeline/MGCB.exe') ">/opt/monogame-pipeline/MGCB.exe</MonoGameContentBuilderExe>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <TargetFrameworkProfile />
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Unfortunately, I abused our repo with some personal git experimentation.  I'll try to not do that in the future.  The transition from TFS to Git is much more difficult since I transitioned from Git to TFS 2 years ago.

I thought I explained in the comments why I closed #99.   I'll see what I can do about correcting this.  Pull requests do not work the way I thought I understood them.

Hero6.DesktopGL uses the .Net 4.5.1 framework.